back/boxtribute_server/business_logic/statistics/sql.py

```diff
@@ -306,6 +336,10 @@
     TRIM(LOWER(p.name)) AS product_name,
     p.gender_id AS gender,
     d.source_size_id AS size_id,
+    -- neglect possible history of box's measure_value
+    CONCAT(ROUND(b.measure_value, 3),
```

Comment:
move the round function up to the cached function

Reply:
SELECT ROUND(your_decimal_field, 3 - FLOOR(LOG10(ABS(your_decimal_field)) + 1)) AS rounded_value
FROM your_table_name;

Reply:
round to 3 significant non-zero digits

Reply:
AGREED: BE returns absoluteMeasure (just the number rounded to 3 non-zero significant digits) and dimension_id in facts. Dimension_id is then put into dimensions too